src/ProjectTemplates/README.md

```diff
@@ -11,7 +11,7 @@ The following contains a description of each sub-directory in the `ProjectTempla
 - `Web.Client.ItemTemplates`: Contains the Web Client-Side File templates, includes things like less, scss, and typescript
 - `Web.ItemTemplates`: Contains the Web File templates, includes things like: protobuf, razor component, razor page, view import and start pages
 - `Web.ProjectTemplates`: Contains the ASP.NET Core Web Template pack, including Blazor Server, WASM, Empty, Grpc, Razor Class Library, RazorPages, MVC, WebApi.
-- `Web.Spa.ProjectTemplates`: Contains the Single Page Application templates for ASP.NET Core, including Anuglar, React, ReactRedux.
+- `Web.Spa.ProjectTemplates`: Contains the Single Page Application templates for ASP.NET Core, including Anuglar, React, ReactRedux. *Note, that this folder is brought by a submodule from the dotnet/spa-templates repo.**
```

Comment:
Not quite sure how to phrase this because the Web.Spa.ProjectTemplates _folder_ no longer exists anywhere. The project Microsoft.DotNet.Web.Spa.ProjectTemplates.csproj still exists however.

I also suggest moving this into a second paragraph because `contains a description of each sub-directory` doesn't work for this project.